main.py

```diff
@@ -0,0 +1,107 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import schemas
+import crud
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+def read_root() -> dict:
+    return {"Hello": "World"}
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def get_all_authors(db: Session = Depends(get_db)):
+    return crud.get_all_authors(db)
+
+
+@app.get("/skip_authors/", response_model=list[schemas.Author])
+def get_skip_authors(skip_value: int, db: Session = Depends(get_db)):
+    return crud.get_skip_authors(db, skip_value)
+
+
+@app.get("/limit_authors/", response_model=list[schemas.Author])
+def get_limit_authors(limit_value: int, db: Session = Depends(get_db)):
+    return crud.get_limit_authors(db, limit_value)
+
+
+@app.get("/author/", response_model=schemas.Author)

```

Comment:
The default approach is to get values from DB by the id of a row. Not an author's name.